src/shared/lib/FormatUtils.tsx

```diff
@@ -103,20 +103,22 @@ export function toConditionalPrecisionWithMinimum(
     positiveNumber: number,
     precision: number,
     precisionThreshold: number,
-    minimumExponent: number
-) {
+    minimumExponent: number,
+    prependEqualSign?: boolean
+): string | React.ReactNode {
     if (positiveNumber === 0 || Math.log10(positiveNumber) < minimumExponent) {
         return (
             <span>
                 &lt; 10<sup>{minimumExponent}</sup>
             </span>
         );
     } else {
-        return toConditionalPrecision(
+        let transformed = toConditionalPrecision(
             positiveNumber,
             precision,
             precisionThreshold
         );
+        return <>{prependEqualSign ? ' = ' : '' }{transformed}</>;
```

Comment:
seems like the equal sign belongs outside this function.  what is it for?

Reply:
`toConditionalPrecisionWithMinimum` sometimes returns jsx, which makes checking if it starts with an `<` a bit dirty.
For example:
- when `<10^-10` no equal sign has to be added in `p < 10^-10`
- when `1.23e-4` you want to add an equal sign in `p = 1.23^4`
So I added the equal sign functionality to `toConditionalPrecisionWithMinimum`, to avoid having to parse the jsx

But maybe we can also simplify it, by rendering in the plot `p: < 10^-10` and `p: 1.23^4`? In that case no equal sign has to be added at all.

Reply:
i see.  no i think we should keep equal sign.  guess there's no good solution here so this is fine.